style.css

```diff
@@ -1,27 +1,221 @@
-body
-{
-background-color: blue;
- }
-
-.navbar
-{
-  display: inline-block;
-  font-size:60px;
-  padding: 10px;
-}
-.icon
-{
-  height: 8%;
-  width: 8%;
-}
-.grid-container
-{
-  display:grid;
-  grid-template-columns: 1fr 1fr;
-  grid-gap: 20px;
-}
-.thumbnail
-{
-  display:block;
-  margin:2%;
+body {
+	box-sizing: content-box;
+	margin: 0;
+	background-color: whitesmoke;
+	font-family:sans-serif;
+}
+
+i {
+	color: #606060;
+}
+
+iframe {
+	width: 100vw;
+	height: 56.25vw; /* 100/56.25 = 560/315 = 1.778 */
+	background: red;
+	border:0;
+}
+.side-flex {
+	display: flex;
+	flex-direction: column;
+}
+
+.thumb{
+	display:flex;
+	flex-direction: row;
+	height:auto;
+	word-wrap: break-word;
+
+}
+
+.side-title{
+	font-weight: 600;
+	margin:0;
+	font-size:0.8rem;
+}
+
+.side-description{
+		color:rgb(56, 55, 55);
+		margin:4px 0 0 0;
+		font-size:0.8rem;
+}
+
+.side-combo {
+	margin:2%;
+
+}
+
+.views {
+	color: rgb(56, 55, 55);
+}
+
+.subscribe {
+	display: flex;
+	align-items: center;
+	justify-content: flex-end;
+}
+
+#subscribe-button {
+	background-color: rgb(199, 41, 41);
+	color: white;
+	border: none;
+	border-radius: 3px;
+	height: 2.4rem;
+	width: 7rem;
+	text-transform: uppercase;
+	font-family: sans-serif;
+}
+
+.grid-container {
+	width: 96%;
+	padding-top: 5rem;
+	padding-left: 1.5rem;
+	display: grid;
+	grid-template-columns: 66% 34%;
+	grid-gap: 1.2rem;
+}
+
+.small-grid {
+	top: 0;
+	width: auto;
+	display: grid;
+	grid-gap: 0.5rem .1rem;
+	grid-template-columns: 1.4fr 1fr;
+}
+
+#videotitle {
+	font-size: 30px;
+	margin-bottom: 1rem;
+}
+
+.description {
+	padding-top: 54%;
+	font-size: 20px;
+}
+
+#account_icon {
+	display: grid;
+	font-size: 60px;
+	align-content: center;
+	justify-content: center;
+}
+.account {
+	display: grid;
+	grid-template-columns: 1fr 5fr;
+	grid-gap: 0;
+}
+
+.social {
+	color: #606060;
+	font-size: 1rem;
+	display: flex;
+	align-items: center;
+	justify-content: flex-end;
+}
+
+.social-icon {
+	padding-right: 0.8rem;
+	padding-left: 0.4rem;
+}
+
+.mainvideo {
+	position: relative;
+	width: 100%;
+	height: 0;
+	padding-bottom: 56%;
+}
+
+.mainvideo iframe {
+	position: absolute;
+	width: 100%;
+	height: 100%;
+	margin-left: 0;
+	margin-top: 0;
+}
+
+.navbar {
+	background-color: white;
+	display: flex;
+	font-size: 30px;
+	justify-content: space-between;
+	position: fixed;
+	width: 100%;
+	z-index: 1;
+}
+
+.navElement {
+	display: flex;
+	padding: 5px;
+}
+
+.logo {
+	height: 3rem;
+	padding-left: 1rem;
+}
+
+.thumbnail {
+	display: flex;
+	margin: 2%;
+	max-width: 10rem;
+	height:56%;
+	width:100%;
+}
+
+.notification {
+	/* padding-top: 0.25rem; */
```

Comment:
Remove the commented code.

❌ commented piece of codes `/* .nav { color: red;} */`
✔ comments about code /*Navigation*/